src/os/ui/ol/interaction/mousewheelzoominteraction.js

```diff
@@ -1,105 +1,43 @@
-      var map = mapBrowserEvent.map;
-      var wheelEvent = /** @type {WheelEvent} */ (mapBrowserEvent.originalEvent);
-
-      if (this.useAnchor_) {
-        this.lastAnchor_ = mapBrowserEvent.coordinate;
-      }
-
-      // Delta normalisation inspired by
-      // https://github.com/mapbox/mapbox-gl-js/blob/001c7b9/js/ui/handler/scroll_zoom.js
-      // TODO There's more good stuff in there for inspiration to improve this interaction.
-      var delta;
-      if (mapBrowserEvent.type == ol.events.EventType.WHEEL) {
-        // treat horizontal scroll the same as normal scroll
-        delta = wheelEvent.deltaY || wheelEvent.deltaX;
-
-        if (ol.has.FIREFOX &&
-            wheelEvent.deltaMode === goog.global.WheelEvent.DOM_DELTA_PIXEL) {
-          delta /= ol.has.DEVICE_PIXEL_RATIO;
-        }
-        if (wheelEvent.deltaMode === goog.global.WheelEvent.DOM_DELTA_LINE) {
-          delta *= 40;
-        }
-      } else if (mapBrowserEvent.type == ol.events.EventType.MOUSEWHEEL) {
-        delta = -wheelEvent.wheelDeltaY;
-        if (ol.has.SAFARI) {
-          delta /= 3;
-        }
-      }
-
-      this.delta_ += delta;
+os.ui.ol.interaction.MouseWheelZoom = function(opt_options) {
+  var options = opt_options || {};
 
-      if (this.startTime_ === undefined) {
-        this.startTime_ = Date.now();
-      }
+  // change the default zoom timeout period so it feels less sluggish
+  options.timeout = options.timeout != null ? options.timeout : 10;
 
-      var duration = 10;
-      var timeLeft = Math.max(duration - (Date.now() - this.startTime_), 0);
+  // disable animation by default
+  options.duration = options.duration != null ? options.duration : 0;
 
-      goog.global.clearTimeout(this.timeoutId_);
-      this.timeoutId_ = goog.global.setTimeout(this.doZoom_.bind(this, map), timeLeft);
-    }
+  os.ui.ol.interaction.MouseWheelZoom.base(this, 'constructor', opt_options);
 
-    mapBrowserEvent.preventDefault();
-    stopEvent = true;
-  }
-  return !stopEvent;
+  // the default value feels sluggish. this was observed on macOS and may need to be fine tuned for other environments
+  this.trackpadDeltaPerZoom_ = 100;
 };
+goog.inherits(os.ui.ol.interaction.MouseWheelZoom, ol.interaction.MouseWheelZoom);
 
 
 /**
- * Override to remove the zoom animation and lock the zoom to increments of 0.2 zoom levels.
- * @param {ol.PluggableMap} map Map.
- * @private
- *
- * @suppress {accessControls|duplicate} Replacing a private function from OL3.
+ * @inheritDoc
+ * @suppress {accessControls} Replacing a private function from OL3.
  */
-ol.interaction.MouseWheelZoom.prototype.doZoom_ = function(map) {
-  var view = map.getView();
-  if (view) {
-    var delta = this.delta_ > 0 ? 0.2 : -0.2;
-    map.render();
-    ol.interaction.Interaction.zoomByDelta(view, -delta, this.lastAnchor_);
+os.ui.ol.interaction.MouseWheelZoom.prototype.handleWheelZoom_ = function(map) {
+  if (this.delta_) {
```

Comment:
We can now override the animation duration (to 0), so we just have to tweak the delta.